### ISSUE TITLE
tried using it.. does not seem to work!

### ISSUE THREAD

**Issue by artknight** [trentrichardson/CaldavCFC#1]
Hi,
I just tried using your component but it does not seem to be working! Upon calling the URL the component throws an error however I cannot figure out what the error is.. it just returns the host in the error message.

<img width="1290" alt="Screen Shot 2022-04-20 at 1 17 24 PM" src="https://user-images.githubusercontent.com/157762/164286810-7bf454d2-4fee-44e3-aca1-ce3592fbaa04.png">

**Comment by trentrichardson** [comment]
I've not run it before on Lucee, so there could be an issue there. Only
thing I see in your screenshot resembling an error is Unknown host
exception. Can the server you are running this from access the URL?

**Comment by artknight** [comment]
Yes, even when I put the URL in the browser I get the login popup.

**Comment by artknight** [comment]
this is how I am instantiating it

```
var __caldav = new lib.caldav(
	host = arguments.integration.params.server_url,
	port = 443,
	path = "",
	username = arguments.integration.params.username,
	password = arguments.integration.params.password
);
```

**Comment by trentrichardson** [comment]
Could it possibly be the port number? I'm sorry I'm not much help, I've not
written a line of cold fusion in years, and about 10 years since I created
this. Beyond that you could go into the source and put a cfdump just before
the makerequest function to ensure all your variables are as you expect.

**Comment by artknight** [comment]
ok, I think i figured it out :)

It seems the `host` and `path` need to be set correctly. My **host** was `https://site.com/cals/123` and it needed to be just `site.com`, similarly my **path** was `""` and it needed to be `cals/123`.

I am now getting **true** for `socket.isConnected()`. 

Thank you!!
Art

**Comment by trentrichardson** [comment]
Awesome! Glad you got it going.

**Comment by artknight** [comment]
well, celebrated too soon :(

Do you think you could you try to get it connected? I am not sure what is wrong. About to pull my hair out.. lol

I have setup a test account and checked from a different app to make sure it is working!

```
server_url: https://email.praxis.my/SOGo/dav/test-demo@praxis.my/Contacts/personal
username: test-demo@praxis.my
password: V7d7HC4Kvxo82Y
```

Do you think you could give it a try?
